ipaserver/advise/plugins/smart_card_auth.py

```diff
@@ -315,7 +315,8 @@ def add_pkcs11_module_to_systemwide_db(self):
 
     def run_authconfig_to_configure_smart_card_auth(self):
         self.log.exit_on_failed_command(
-            'authconfig --enablesmartcard --smartcardmodule=sssd --updateall',
+             'authconfig --enablesssd --enablesssdauth --enablesmartcard'
```

Comment:
You are missing a space at the end of the line. Right now the line will print as
``... --enablesmartcard--smartcardmodule=sssd ...``
instead of
``... --enablesmartcard --smartcardmodule=sssd ...``